blackbox/docs/general/dml.rst

```diff
@@ -451,9 +451,31 @@ used::
 
 This wildcard can also be used to only match certain files::
 
-    cr> copy quotes from '/tmp/import_data/qu*.json';
+    cr> COPY quotes FROM '/tmp/import_data/qu*.json';
     COPY OK, 3 rows affected (... sec)
 
+.. Hidden: delete imported data
+
+    cr> refresh table quotes;
+    REFRESH OK, 1 row affected (... sec)
+    cr> delete from quotes;
+    DELETE OK, 3 rows affected (... sec)
+    cr> refresh table quotes;
+    REFRESH OK, 1 row affected (... sec)
+
+Using the ``RETURN SUMMARY`` clause, a result set containing detailed
+information of possible failures and successfully imported records.
```

Comment:
How about:

If the ``RETURN_SUMMARY`` clause is specified, a result set containing information about failures and successfully imported records is returned.